assets/js/state/sagas/hosts.test.js

```diff
@@ -0,0 +1,18 @@
+import { recordSaga } from '@lib/test-utils';
+import { hostDeregistered } from '@state/sagas/hosts';
+import { HOST_DEREGISTERED, removeHost } from '@state/hosts';
+import { hostFactory } from '@lib/test-utils/factories';
+
+describe('Hosts sagas', () => {
+  it('should remove the host', async () => {
+    const { id, hostname } = hostFactory.build();
+    const payload = { id, hostname };
+
+    const dispatched = await recordSaga(hostDeregistered, {
+      type: HOST_DEREGISTERED,
```

Comment:
Why we need the type field here>
I think we don't need the type value here, we could just easily add the payload like the other tests